app/src/main/res/layout/fragment_dummy.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".DummyFragment">
+
+    <!--TODO use extracted preset padding values in dimens.xml-->
```

Comment:
Master already has the preset values that you can start using

Reply:
I used the preset values already in both this `fragment_dummy.xml` and `item_category.xml`. I think I forgot to do `git rebase --continue` before I committed the changes... so the layout files on my local branch already have the preset values in use and committed. I'll see if I can resolve this!